lib/raven/interfaces/http.rb

```diff
@@ -15,6 +15,7 @@ class HttpInterface < Interface
     def initialize(*arguments)
       self.headers = {}
       self.env = {}
+      self.cookies = nil
       super(*arguments)
```

Comment:
I don't think so. I can't recall immediately but removing better_attr_accessor did cause this test to fail.